src/test/java/hw2/ex1/DefaultPageTest.java

```diff
@@ -0,0 +1,124 @@
+package hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DefaultPageTest {
+
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void ChromeSetUp()
+    {
+        driver = new ChromeDriver();
+    }
+
+    @Test (description = "Default Page Test")
+    public void MyTes_01() {
+
+        driver.manage().window().maximize();
+
+        //1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
```

Comment:
Maybe we should use a string constant for site URL?

Reply:
string constant added